README.md

```diff
@@ -25,10 +25,37 @@ The release page includes the full Changelog but you may also see the [changelog
 
 ## Build requirements
 * Visual Studio 2017 or newer
-* optional: CMake (when building with CMake)
+* optional: CMake (when building with CMake), Ninja
 
 ## HowTo
 You may use win_flex and win_bison directly on the command line or [use them via CustomBuildRules in VisualStudio](custom_build_rules/README.md).
+### Visual Studio Code
+Alternatively, VS Code is available for debugging, here is a simple guide (based on the [MS documentation](https://code.visualstudio.com/docs/cpp/cmake-linux)):
+1. Make sure the [Prerequisites](https://code.visualstudio.com/docs/cpp/cmake-linux#_prerequisites) are installed.
+2. [Select a compiler kit](https://code.visualstudio.com/docs/cpp/cmake-linux#_select-a-kit), any of the native `amd64` version of MSVC is recommended for compatibility - any of the rest with `x86` is cross-compiling a 32-bit version of `win_flex`/`win_bison`.
+
+![Select a compiler kit](.vscode/1_compiler.png)
+
+3. Due to `win_flex`/`win_bison` being separate executables/targets, debugging will need to [select a launch target](https://github.com/microsoft/vscode-cmake-tools/blob/main/docs/debug-launch.md#select-a-launch-target).
+
+![Select a launch targe](.vscode/2_target.png)
+
+4. Optionally switch between build types - usually between `Release` or `Debug` builds, make sure `Debug` is selected when debugging or there will be no debug symbols to hit breakpoints.
+
+![Debug build type](.vscode/3_build.png)
+
+5. Both `win_flex` and `win_bison` requires arguments for the target input/output source files, just add the arguments as how `win_flex` or `win_bison` was called from the command line where each space-separated argument is an element in [.vscode/launch.json](.vscode/launch.json)'s `args` array separated by commas, here is a sameple for `win_flex` target:
+```
+            "args": [
+                "--header-file=sample.tab.h",
+                "-osample.tab.c",
+                "sample.l"
+            ],
+```
+
+6. Setup the IDE for inspection [debugging](https://code.visualstudio.com/docs/editor/debugging), eg. breakpoints, logpoints.
```

Comment:
it likely is more reasonable to not use "IDE" here; not sure if "editor" as used in the vscode docs is the best option, but it would be better.

Wording/spacing/punctuation:  "Set up" and "e.g. breakpoints or logpoints."

Reply:
How about calling it "codebase"? It'll be vague but indicative

Reply:
I'm fine with that.